crates/bevy_scene/src/lib.rs

```diff
@@ -40,9 +41,7 @@ impl Plugin for ScenePlugin {
             .add_asset::<Scene>()
             .init_asset_loader::<SceneLoader>()
             .init_resource::<SceneSpawner>()
-            .add_systems(Update, scene_spawner_system)
-            // Systems `*_bundle_spawner` must run before `scene_spawner_system`
-            .add_systems(PreUpdate, scene_spawner);
+            .add_systems(PostUpdate, (scene_spawner, scene_spawner_system).chain());
```

Comment:
I'm really sorry, but could you move it back to `PreUpdate`? Looks like `PostUpdate` is more controversial because it will require explicit ordering and it will be harder to react on scene spawn since It will require order "reaction" system between transform propagation and scene spawn.
```suggestion
            .add_systems(PreUpdate, (scene_spawner, scene_spawner_system).chain());
```
`PreUpdate` also works just fine. @james7132 what do you think?

Reply:
Spawning new things in PostUpdate feels wrong because PostUpdate is "arbitrary reaction logic to world data that runs before rendering". This isn't just about transform propagation. It is an arbitrary / unknowable class of thing
We can't afford to miss any of it
Imo we shouldn't do scene spawning "midway" through any stage as that makes it hard to reason about / makes the behavior undefined. These are the valid places for it imo:

```
1. [SpawnScene][PreUpdate][Update][PostUpdate]
2. [PreUpdate][SpawnScene][Update][PostUpdate]
3. [PreUpdate][Update][SpawnScene][PostUpdate]
4. [PreUpdate][Update][PostUpdate][SpawnScene]
```


And (4) should not be considered because it would result in rendering things without the PostUpdate reactions
(2) vs (3) is 

2) "do we want to run Update logic for a scene before rendering it?"
3) "do we want to spawn scenes on the frame they are spawned?"

Reply:
I personally like 3 more because it also removes one frame delay between `Handle<Scene>` insertion.